src/Benchmarks.ServerJob/ServerJob.cs

```diff
@@ -60,6 +60,7 @@ public class ServerJob : IIdentifiable
         public string RuntimeVersion { get; set; } = "";
         public string SdkVersion { get; set; } = "";
         public bool UseMonoRuntime { get; set; } = false;
+        public bool UseMonoRuntime_LlvmJit = false;
```

Comment:
Maybe merge these two fields in one `string` containing the type of jit to use. I assume there might be a third value at some point. The option can still have no value to be compatible with the existing command line argument.

Reply:
Yes, there will be a third flavor, which is AOT LLVM. What do you mean by merging those two fields but the option could still have no value? If we merge them, then it has to have a value to differentiate which flavor the use wants to have, right?

Reply:
I got it now...